python/graphscope/client/session.py

```diff
@@ -1392,17 +1392,19 @@ def nx(self):
             return self._nx
         import importlib.util
 
-        spec = importlib.util.find_spec("graphscope.nx")
-        mod = importlib.util.module_from_spec(spec)
-        spec.loader.exec_module(mod)
-
-        graph = type("Graph", (mod.Graph.__base__,), dict(mod.Graph.__dict__))
-        digraph = type("DiGraph", (mod.DiGraph.__base__,), dict(mod.DiGraph.__dict__))
-        setattr(graph, "_session", self)
-        setattr(digraph, "_session", self)
-        setattr(mod, "Graph", graph)
-        setattr(mod, "DiGraph", digraph)
-        self._nx = mod
+        try:
+            spec = importlib.util.find_spec("graphscope.nx")
+            mod = importlib.util.module_from_spec(spec)
+            spec.loader.exec_module(mod)
+            graph = type("Graph", (mod.Graph.__base__,), dict(mod.Graph.__dict__))
+            digraph = type("DiGraph", (mod.DiGraph.__base__,), dict(mod.DiGraph.__dict__))
+            setattr(graph, "_session", self)
+            setattr(digraph, "_session", self)
+            setattr(mod, "Graph", graph)
+            setattr(mod, "DiGraph", digraph)
+            self._nx = mod
+        except ModuleNotFoundError:
```

Comment:
no need to add `try `here, since the ModuleNotFoundError is the error we want to raise.